playbooks/openshift/post_install.yml

```diff
@@ -1,15 +1,43 @@
 - name: Run NFS volume creation
   include: ../../../openshift-ansible-tourunen/setup_lvm_nfs.yml
 
-- name: Copy files to master-1
+- name: Import objects through master-1
```

Comment:
While the reference to master-1 makes sense here, it could be changed to make sense with Heat provisioning later on. It can be a bit confusing since what is really meant here is "first master". But Heat provisioning is currently in a feature branch and not part of master, so not really a reason to not merge.